assets/src/apps/authoring/components/EditingCanvas/EditingCanvas.tsx

```diff
@@ -126,6 +132,39 @@ const EditingCanvas: React.FC = () => {
     dispatch(setRightPanelActiveTab({ rightPanelActiveTab: RightPanelTabs.SCREEN }));
   }, [currentActivityId]);
 
+  useKeyDown(
+    () => {
+      if (currentSelectedPartId && !configPartId?.length && _currentPartPropertyFocus) {
+        setNotificationStream({
+          stamp: Date.now(),
+          type: NotificationType.CHECK_SHORTCUT_ACTIONS,
+          payload: { id: currentSelectedPartId, type: 'Delete' },
+        });
+      }
+    },
+    ['Delete', 'Backspace'],
+    {},
+    [currentSelectedPartId, configPartId, _currentPartPropertyFocus],
+  );
+
+  useKeyDown(
+    () => {
+      if (currentSelectedPartId && !configPartId?.length && _currentPartPropertyFocus) {
```

Comment:
For '`Cntrl+C`', only trigger if the focus is on that selected part. There are use cases where, user can select a part and tries to copy the text or Id. In these cases, we should not trigger the part copy.